apps/condo/migrations/20240725131845-0409_marketsettinghistoryrecord_and_more.js

```diff
@@ -0,0 +1,98 @@
+ALTER TABLE "OrganizationEmployeeRole" ALTER COLUMN "canManageMarketSetting" DROP DEFAULT;
+--
+-- Add field canReadMarketSetting to organizationemployeerole
+--
+ALTER TABLE "OrganizationEmployeeRole" ADD COLUMN "canReadMarketSetting" boolean DEFAULT false NOT NULL;
+ALTER TABLE "OrganizationEmployeeRole" ALTER COLUMN "canReadMarketSetting" DROP DEFAULT;
+--
+-- Add field canManageMarketSetting to organizationemployeerolehistoryrecord
+--
+ALTER TABLE "OrganizationEmployeeRoleHistoryRecord" ADD COLUMN "canManageMarketSetting" boolean NULL;
+--
+-- Add field canReadMarketSetting to organizationemployeerolehistoryrecord
+--
+ALTER TABLE "OrganizationEmployeeRoleHistoryRecord" ADD COLUMN "canReadMarketSetting" boolean NULL;
+--
+-- Create model marketsetting
+--
+CREATE TABLE "MarketSetting" ("residentAllowedPaymentTypes" jsonb NOT NULL, "id" uuid NOT NULL PRIMARY KEY, "v" integer NOT NULL, "createdAt" timestamp with time zone NULL, "updatedAt" timestamp with time zone NULL, "deletedAt" timestamp with time zone NULL, "newId" uuid NULL, "dv" integer NOT NULL, "sender" jsonb NOT NULL, "createdBy" uuid NULL, "organization" uuid NOT NULL, "updatedBy" uuid NULL);
+--
+-- Create constraint MarketSetting_unique_organization on model marketsetting
+--
+CREATE UNIQUE INDEX "MarketSetting_unique_organization" ON "MarketSetting" ("organization") WHERE "deletedAt" IS NULL;
+CREATE INDEX "MarketSettingHistoryRecord_history_id_cfb09a90" ON "MarketSettingHistoryRecord" ("history_id");
+ALTER TABLE "MarketSetting" ADD CONSTRAINT "MarketSetting_createdBy_599c8421_fk_User_id" FOREIGN KEY ("createdBy") REFERENCES "User" ("id") DEFERRABLE INITIALLY DEFERRED;
+ALTER TABLE "MarketSetting" ADD CONSTRAINT "MarketSetting_organization_115b6aff_fk_Organization_id" FOREIGN KEY ("organization") REFERENCES "Organization" ("id") DEFERRABLE INITIALLY DEFERRED;
+ALTER TABLE "MarketSetting" ADD CONSTRAINT "MarketSetting_updatedBy_956056cd_fk_User_id" FOREIGN KEY ("updatedBy") REFERENCES "User" ("id") DEFERRABLE INITIALLY DEFERRED;
+CREATE INDEX "MarketSetting_createdAt_769f0e13" ON "MarketSetting" ("createdAt");
+CREATE INDEX "MarketSetting_updatedAt_de870832" ON "MarketSetting" ("updatedAt");
+CREATE INDEX "MarketSetting_deletedAt_07e3a1a2" ON "MarketSetting" ("deletedAt");
+CREATE INDEX "MarketSetting_createdBy_599c8421" ON "MarketSetting" ("createdBy");
+CREATE INDEX "MarketSetting_organization_115b6aff" ON "MarketSetting" ("organization");
+CREATE INDEX "MarketSetting_updatedBy_956056cd" ON "MarketSetting" ("updatedBy");
+--
+-- [CUSTOM] Set Statement Timeout to some large amount - 25 min (25 * 60 => 1500 sec)
+--
+SET statement_timeout = '1500s';  
+UPDATE "OrganizationEmployeeRole"
+SET "canManageMarketSetting" = true,
+    "canReadMarketSetting" = true
+WHERE "name" = 'employee.role.Administrator.name';
+--
+-- [CUSTOM] Revert Statement Timeout to default amount - 10 secs
+--
```

Comment:
You should updated rules (read: true) for Dispatcher, Foreman and Technician.

This is indicated in the default values